otx/algorithms/classification/configs/base/data/cls_incr_data_pipeline.py

```diff
@@ -0,0 +1,45 @@
+"""Data Pipeline of Class-Incr model for Classification Task."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+# pylint: disable=invalid-name
+
+img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
```

Comment:
why don't you use `__img_norm_cfg` instead of `img_norm_cfg`?

Reply:
That's a great suggestion! In line with this, i will go through and modify all the variables in `data_pipeline.py`.

Reply:
Edited as suggested.